### DISCUSSION TITLE
cocos2d-x or axmol engine?

### DISCUSSION THREAD

**Discussion by 10100010010001001010** [axmolengine/axmol#1056] [General]
Hi, please tell me, is it worth learning cocos2d-x or should I start with axmol instead? How is the original inferior to the fork version?

It is also unclear whether AXIS AXYS AXDE is the same, or are they different branches?

**Comment by halx99** [comment]
The axis, axys, adxe is older name of axmol, recommand use axmol instead cocos2dx-4.0

**Comment by DelinWorks** [comment]
The question should be "should I learn cocos2dx/axmol" because they're the same! a C++ game engine with the same architecture except that axmol is maintained, refined, and more up to date.

**Comment by 10100010010001001010** [comment]
What about the API reference? As i understood i should using original API reference for unchanged functions and using axmol API reference for new/changed functions?

**Reply by DelinWorks** [reply]
there has been some few changes to the api if I'm not mistaken, you could look into major ones in [RENAMED_TYPES.md](https://github.com/axmolengine/axmol/blob/dev/RENAMED_TYPES.md) other than that it's enhancements, like video playback webview2 google angle and so much more, you can also port your cocos2dx project to this engine with some minor changes
